### DISCUSSION TITLE
ΓΕ5 - Μέρος Α - Συνεννόηση για την επιλογή βιογραφίας/μελέτης περίπτωσης

### DISCUSSION THREAD

**Discussion by afoscolou** [sdy60-2022/.github#46] [General]
Αν και είναι πολύ πρώιμο, επειδή ανακοινώθηκε η εκφώνηση της 5ης εργασίας, ανοίγω καινούριο thread για τον μεταξύ μας συντονισμό  ως προς τις βιογραφίες/μελέτες περίπτωσης που θα διαλέξουμε. Όποιος βρίσκει τι θέλει να χρησιμοποιήσει, μπορεί να το ποστάρει σε αυτή την συζήτηση προς ενημέρωση των υπολοίπων.

**Comment by vkaragia** [comment]
Καλημέρα, θα γράψω μια περιπτωσιολογική μελέτη με θέμα "Enabling People with Visual Impairments to Navigate Virtual Reality with a Haptic and Auditory Cane Simulation"

**Comment by afoscolou** [comment]
Καλησπέρα, επιλέγω τη μελέτη περίπτωσης "Aspen Movie Map".

**Comment by sotkot** [comment]
Επιλέγω τη μελέτη περίπτωσης "The LEEP Panoramic Stereo Photography System - LeepVR"

**Comment by gregtsag** [comment]
Βιογραφία για τον Don Norman

**Comment by atsiano81** [comment]
θα γράψω λίγα λόγια για bci (brain computer interfaces) σαν μέρος του hci
